### PR TITLE
Registration with template

### PR DESCRIPTION
**Pull request by xzwache**

### CODE REVIEW

File: app/forms.py [hunk 1]
```diff
@@ -1 +1,51 @@
 # Here will be Flask Web Forms
```
Comment (CHuKeR) [inline app/forms.py]: pep8 fix for all file please (a lot of excess empty lines)

File: app/routes.py [hunk 2]
```diff
@@ -1,9 +1,34 @@
...
 
 
 @app.route('/')
 def index():
     feed_blank = 'Podcast Main page: RSS feed'
     return render_template('index.html', feed_blank=feed_blank)
+
+
+@app.route('/')
+def login():
```
Comment (CHuKeR) [inline app/routes.py]: second '/' route?

**Review by CHuKeR — CHANGES_REQUESTED** [review]
read in comments

File: app/routes.py (hunk 2 above)
Comment (xzwache) [reply app/routes.py]: This if only for tests. Sorry about that.

**Review by CHuKeR — APPROVED** [review]

File: app/forms.py [hunk 3]
```diff
@@ -7,33 +7,29 @@
...
     confirm_password = PasswordField(
         'Подтверждение пароля',
         validators=[
             DataRequired(),
             Length(min=3, max=25),
             EqualTo('password')])
-
     submit = SubmitField('Зарегистрироваться')
 
     @staticmethod
```
Comment (CHuKeR) [inline app/forms.py:35]: I don't understand, what are these methods for?
And, if you use decorator staticmethod, you shouldn't get 'self' param.

**Review by CHuKeR — APPROVED** [review]

**Pull request merged**